src/game/overworld-ui/movement.js

```diff
@@ -0,0 +1,78 @@
+const globals = require('../../game-data/globals')
+
+let frameCount = 0;
+let playerCharacter;
+let pressedKeys = {};
+
+const initMovement = (character) => {
+    playerCharacter = character
+    window.requestAnimationFrame(movementController)
+}
+
+// this function gets called by Window.requestAnimationFrame about 60 times per second
+// The pressedKeys variable is manipulated in GfxContainer with Eventlisteners
+
+const movementController = ( ) => {    
+    
+    let frontContext = playerCharacter.ctx
+        
+    frontContext.clearRect( 
+        playerCharacter.xy.x, playerCharacter.xy.y, 
```

Comment:
kan ff niet vinden waar dat character model staat maar is het handig om xy een object met een x en een y te maken, waaorm niet gwn een x en een y in het playerCharacter object. Of geef een propname als location ofzo of gridlocation en dan een x en een y